extensions/facets/standard/clause_7_simple_facets.adoc

```diff
@@ -90,22 +100,26 @@ TODO: clarify `more` data type (integer? boolean?  Both)?
     "facets": {
         "keywords": {
             "type": "terms",
-            "property": "keyword",
+            "property": "tag",
             "buckets": [
                 {
-                    "value": "Greece",
+                    "value": "Environment",
                     "count": "202"
                 },
                 {
-                    "value": "Germany",
+                    "value": "Atmosphere",
                     "count": "150"
+                },
+                {
+                    "value": "Open-data",
+                    "count": "47"
                 }
             ],
-            "more": 0
+            "more": false
```

Comment:
We should update the associated schema in https://github.com/opengeospatial/ogcapi-records/blob/master/extensions/facets/openapi/schemas/facets-items.yaml#L56